PRODSECBUG-2347 Release Note Update (#4103)
magento/devdocs: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md

6. Refresh the cache from the Admin (**System** > **Cache Management**).

## PayPal Payflow Pro active carding activity update

> The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.

Reviewer: The changes are on the form itself to use reCAPTCHA/CAPTCHA. They are not updating the modules. Suggested edit:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```
